src/components/common/CodeReview.js

```diff
@@ -335,13 +347,19 @@ class CodeReview extends Component {
           line: lineNum,
           ...textContent,
           parent: parentId,
+          codeId,
         },
       },
     ];
     this.setState({
       comments: newComments,
       focusedId: NEW_COMMENT_ID,
     });
+    // make sure the comments on that line are not hidden
+    if (lineCommentsHiddenState[lineNum - 1]) {
```

Comment:
Maybe add comment saying that `lineNum` is 1-indexed while the underlying line is 0-indexed, if that's correct.